### ISSUE TITLE
Debugger controls are invisible

### ISSUE THREAD

**Issue by ericawright** [devtools-html/debugger.html#7240]
![screen shot 2018-11-06 at 9 57 09 am](https://user-images.githubusercontent.com/10803178/48072532-bb6d6a00-e1aa-11e8-9b4a-971a67b30930.png)

This is on Firefox Nightly 65.0. Dark and light themes.

Open debugger panel, look in the upper right corner above watch expressions, there should be controls visible. When hovering over the places the controls should be, the button gets a hover effect and seems to correctly respond to clicks.

**Comment by jasonLaster** [comment]
yep - this is sadly coming from a platform `mask` css platform change. hoping they have a fix soon...
